examples/2d/cifar_small_sample.py

```diff
@@ -225,15 +225,14 @@ def main():
 
     # Optimizer
     lr = 0.1
+    optimizer = torch.optim.SGD(model.parameters(), lr=lr, momentum=0.9,
+                                        weight_decay=0.0005)
     M = args.learning_schedule_multi
     drops = [60*M,120*M,160*M]
+    scheduler = torch.optim.MultiStepLR(optimizer, drops, gamma=0.2)
     for epoch in range(0, 200*M):
```

Comment:
Hey @vivianwhite can you investigate the original paper and see why @edouardoyallon uses this scaling factor M? (Alternatively Edouard, if you can drop in with whats the intention of this M that would be helpful)

I am confused by the description in the script ([here](https://github.com/kymatio/kymatio/blob/7e4099539c5c59eea8a76edcd5c940489e8d6cee/examples/2d/cifar_small_sample.py#L169)) which states "samples per class". I am under the impression this was mixed up with the previous argument `num_samples`, which also states in it's help text `samples per class`.

Reply:
M is a learning rate multiplier. From Deep Hybrid Networks paper ([here](https://openaccess.thecvf.com/content_ICCV_2017/papers/Oyallon_Scaling_the_Scattering_ICCV_2017_paper.pdf)): 

> We use the same training settings for our baseline, WRN
> 16-8, and our hybrid scattering and WRN-12. The settings
> are the same as those described for CIFAR-10 in the previous 
> section with the only difference being that we apply a
> multiplier to the learning rate schedule and to the maximum
> number of epochs. The multiplier is set to 10,20,100 for the
> 1000,500, and 100 sample case respectively. For example
> the default schedule of 60,120,160 becomes 600,1200,1600
> for the case of 1000 samples and a multiplier of 10. Finally
> in the case of 100 samples we use a batch size of 32 in lieu
> of 128.

Reply:
Good work!